src/data/memberships.js

```diff
@@ -4,30 +4,30 @@ export default {
     destinationID: 'room1',
     destinationType: 'room',
     members: [
-      {personID: 'user1'},
-      {personID: 'user2'},
-      {personID: 'user3'},
-      {personID: 'user4'},
+      {personID: 'user1', isInMeeting: true},
```

Comment:
actually, `isInMeeting` only makes sense for meetings, so let's only add this field for meetings. And it would be nice to have at least one meeting where all the members are active (in meeting).

Reply:
@cipak, member objects must be consistent, so all properties should exists and have a value

Reply:
ok, but a "room member" and a "meeting member" are like different subclasses of a common "member" class. From this point of view, can they have different/specific fields? `muted` and `isInMeeting` don't make much sense for members of a room. If we must give them a value, though, I would use `undefined` or `null` (expressing a N/A) instead of `false`, what do you think?

Reply:
I think for now having this subclassing will just complicate things and we should keep it as one member object. I think `false` will make it easier to use when checking for values, but I'm ok with it using `null` if you want to denote a difference.

FYI, in general for this codebase, we should use `null` over `undefined` IF we are assigning values to denote that the lack of value is intentional. `undefined` is ok, if we have never assigned a value.